changelogs/fragments/537-logging-severity-bugfix.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - sonic_logging - Addressing bug found in https://github.com/ansible-collections/dellemc.enterprise_sonic/issues/508 where a traceback is thrown if the "severity" value does not exist on the device. (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/537).
```

Please correct this description to state that the cause of the problem was an input playbook that specified one or more logging options, but did not specify a "severity" option. (That is the case where the problem was occurring due to the Ansible infrastructure behavior of auto-filling any unspecified options from the argspec if one or more options are specified. (It is not related to the current device configuration.)

```suggestion
  - sonic_logging - Addressing bug found in https://github.com/ansible-collections/dellemc.enterprise_sonic/issues/508 where a traceback is thrown if the "severity" value is not specified in the incoming playbook or if the incoming playbook specifies a 'severity' value of None. (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/537).
```